platform/ext/target/nordic_nrf/common/nrfx_config.h

```diff
@@ -54,6 +54,10 @@
 #define NRFX_UARTE3_ENABLED 1
 #endif
 
+#if (__DOMAIN_NS == 1)
+#define NRF_TRUSTZONE_NONSECURE
```

Comment:
Hm, I wonder, if this could be set as a cmake variable, as we do in Zephyr.
@oyvindronningstad ^^

Reply:
NRF_TRUSTZONE_NONSECURE? Sure, but I'm not sure why we want to, can you elaborate?

Reply:
We want to, for example, because this macro is used also in `system_*.c` files, which are compiled without processing this file (they don't include `<nrfx.h>`, only `<nrf.h>`). Currently nothing from those files is used in non-secure images, but...

Reply:
Could they just use __DOMAIN_NS instead?

Reply:
No, because they are imported from nrfx/MDK.

Reply:
Ok, I would think just adding `#include <nrfx.h>` would be cleaner, but I'm probably missing something. I will stop now :).

Reply:
@anangl ^^ 
Also wondering if including nrfx.h could be better, pls, clarify :)

Reply:
The thing is that the files that would need to include `<nrfx.h>`:
- `system_nrf5340_application.c`
- `system_nrf9160.c`

come from MDK that is imported with nrfx. It would not be good to modify them in TF-M.

Reply:
got it now, thanks